source/sync/data-model/update-data-model/migrate-schema-partner-collection.txt

```diff
@@ -18,46 +18,28 @@ Manually Update a Data Model
    :depth: 2
    :class: singlecol
 
-This page describes how to manually update a data model. It applies to App Services Apps
-that do not support automatic updates through Development Mode or with schema versioning. For information on how
-to automatically update a data model, refer to :ref:`synced-schema-overview`.
+This page describes how to make breaking changes to your Device Sync data model.
```

Comment:
Note to reviewer: this page will be updated in following PR to centralize info for applying breaking changes (outside of Dev Mode)